### ISSUE TITLE
[suggestion] decrese iron cabinet and managers mining level?

### ISSUE THREAD

**Issue by MSandro** [AshIndigo/Filing_Cabinet#9]
Hello,
I think the HarvestTool is set a little too high. Or was it really intended that the Iron Cabinet and the manager can only be harvested with Iron Pickaxe?

Please fix this for 1.16.5 soon,
thanks

**Comment by AshIndigo** [comment]
Yeah didn't mean to set them that high if I did. I'll go crank that down a level once I am done travelling. Should be fixed by tomorrow or so.

**Comment by AshIndigo** [comment]
Sorry about not being able to post the update

Between university starting back up and my main dev machine having severe issues I haven't been really able to mod properly as of late

Haven't forgotten about this issue
